times.py

```diff
@@ -4,7 +4,10 @@
 def time_range(start_time, end_time, number_of_intervals=1, gap_between_intervals_s=0):
     start_time_s = datetime.datetime.strptime(start_time, "%Y-%m-%d %H:%M:%S")
     end_time_s = datetime.datetime.strptime(end_time, "%Y-%m-%d %H:%M:%S")
+    # I think that it is easier to understand use my below equation:
+    # d = ((end_time_s - start_time_s).total_seconds() - gap_between_intervals_s * (number_of_intervals - 1)) / number_of_intervals
     d = (end_time_s - start_time_s).total_seconds() / number_of_intervals + gap_between_intervals_s * (1 / number_of_intervals - 1)
```

Comment:
In what way do you think it's easier to understand your equation?

Reply:
It's very similar to this [problem](https://leetcode.com/problems/merge-intervals/description/), except this exercise adds some manipulation of time. The equation is `d = ((end_time_s - start_time_s).total_seconds() - gap_between_intervals_s * (number_of_intervals - 1)) / number_of_intervals`